### ISSUE TITLE
Dropbox provider sets email for goth.User.UserID

### ISSUE THREAD

**Issue by kevin-cantwell** [markbates/goth#212]
This line looks incorrect: https://github.com/markbates/goth/blob/v1.45.3/providers/dropbox/dropbox.go#L179

```go
user.UserID = u.Email // Dropbox doesn't provide a separate user ID
```

[According to the documentation](https://www.dropbox.com/developers/documentation/http/documentation#users-get_current_account), the correct field to use here from the raw data should be `account_id`.

**Comment by adamo57** [comment]
is this issue fixed @kevin-cantwell

**Comment by bentranter** [comment]
Gonna close this as fixed in #213, we can re-open if it becomes an issue again.
